pkg/ovsdb/condition.go

```diff
@@ -358,10 +369,16 @@ func (c *Condition) CompareMap(row *map[string]interface{}) (bool, error) {
 		return false, err
 	}
 
-	if (fn == FN_EQ || fn == FN_IN) && isEqualMap(actual, expected) {
```

Comment:
the same comment about a switch

Reply:
sure, it's more elegant with a switch.
Done